### ISSUE TITLE
direction object responses needed for mouse/touch interaction

### ISSUE THREAD

**Issue by zepumph** [phetsims/friction#212]
Currently, it looks like the BookMovementDescriber is only set up to work with PDOM/keyboard interaction. We need to make sure it get's working for mouse/touch too

**Comment by zepumph** [comment]
I tried to do this patch, but it gave the PDOM too many movement alerts:

```diff

Index: js/friction/view/FrictionDragListener.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/js/friction/view/FrictionDragListener.js b/js/friction/view/FrictionDragListener.js
--- a/js/friction/view/FrictionDragListener.js	(revision ae2f4a941a4c130a4f53323f0b72e4a7fb5656b3)
+++ b/js/friction/view/FrictionDragListener.js	(date 1628723941204)
@@ -50,6 +50,8 @@
         const vector = new Vector2( delta.x, delta.y );
 
         model.move( vector );
+
+        bookMovementDescriber.endDrag();
       },
       end: () => {
         model.bottomOffsetProperty.set( 0 );
@@ -59,7 +61,6 @@
 
         // pdom
         temperatureIncreasingDescriber.endDrag();
-        bookMovementDescriber.endDrag();
       },
       tandem: tandem
     } );

```

**Comment by zepumph** [comment]
>Currently, it looks like the BookMovementDescriber is only set up to work with PDOM/keyboard interaction. We need to make sure it get's working for mouse/touch too

This was not really true, we do still get voicing responses, but only after releasing. We should decide when we want to hear these, because if we keep them the way they are now, "Released" overwrites it completely, and we never hear alerts like:
"At top. Move down to rub harder." or "left".

Assigning to designers:

Is it enough to just hear a single response each time you release the book?

**Comment by zepumph** [comment]
Oh, one more question. Above, I thought it would sound much better to make the "Released" alert not interrupt, otherwise we don't get any object responses. I would like to discuss and confirm this behavior.

**Comment by Matthew-Moore240** [comment]
With the keyboard, the "at top" "left" sound pretty good. On the mouse, the responses seem to bunch up and you hear "left" at the very end and it just sounds pointless.

**Comment by zepumph** [comment]
>On the mouse, the responses seem to bunch up and you hear "left" at the very end and it just sounds pointless. 

This will be fixed over in #224
